app/src/main/java/net/skyscanner/backpack/demo/compose/BpkSliderStory.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import net.skyscanner.backpack.compose.slider.BpkRangeSlider
+import net.skyscanner.backpack.compose.slider.BpkSlider
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun BpkSliderStory() {
+  Column(
+    modifier = Modifier.padding(BpkSpacing.Xxl),
+    verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base, Alignment.CenterVertically)
+  ) {
+    DefaultSliderSample()
+    SteppedSliderSample()
+    RangeSliderSample()
+  }
+}
+
+@Composable
+fun RangeSliderSample() {
+  BpkText(text = stringResource(R.string.slider_range), style = TextStyle(fontSize = BpkTheme.typography.label2.fontSize))
```

Comment:
you can just use `BpkTheme.typography.label2` directly as that's already a `TextStyle`

Reply:
You mean without `fontSize =`? Just trying removing =different things here but can't seem to make it work.

Reply:
so on `BpkText` you're setting the `style` by creating a `TextStyle` with specifying the `fontSize` from an existing style. You don't need to create a `TextStyle` at all and can just use `label2` directly

Reply:
Ah! Finally got what you mean! Fixed now!